File: Programmers/src/Javascript/Lv0/인덱스바꾸기.js
function solution(my_string, num1, num2) {
    var answer = '';
    let str = my_string.split("");
    let array = [];
    let tmp = 0;
    
    for(let i=0; i<str.length; i++)
        array.push(str[i]);
    
    tmp = array[num1];
    array[num1] = array[num2];
    array[num2] = tmp;
    
    for(let i=0; i<array.length; i++)
        answer += array[i];
    
    return answer;
}

File: Programmers/src/Java/Lv0/인덱스 바꾸기.java
package Java.Lv0;

class Solution {
    public String solution(String my_string, int num1, int num2) {
        char[] answer = my_string.toCharArray();
        String result = "";
        char temp = 0;
        
        temp = answer[num1];
        answer[num1] = answer[num2];
        answer[num2] = temp;
        
        for(int i=0; i<answer.length; i++)
            result += answer[i];
        
        return result;
    }
}
